packages/core/src/validate.js

```diff
@@ -204,24 +261,31 @@ export default function validateFormData(
   }
 
   let validationError = null;
+
+  // Since ajv@7, schema validation is not automatically performed every time
+  // when validating data, so we explicitly check for schema errors first.
   try {
-    ajv.validate(schema, formData);
+    ajv.validateSchema(schema);
   } catch (err) {
     validationError = err;
   }
 
+  if (validationError === null) {
+    try {
+      ajv.validate(schema, formData);
```

Comment:
ajv v7 changes:
- If you call `ajv.validate` on an invalid schema twice, the second time it will (incorrectly) not throw any errors due to the schema being invalid.

In general, to note about ajv:
- If you call `ajv.validateSchema`, it usually returns false if the schema is invalid. However, if the meta schema is not found, it will throw an exception.

Goal: Go around ajv v7 changes. Aim for same behavior as before (we want to catch all metaschema errors in `validationError`).

Current solution: Call `ajv.validateSchema` beforehand, but regardless of whether the schema is valid, call `ajv.validate`.

New solution: ???